.gitignore

```diff
@@ -29,3 +29,4 @@ docs/_build
 docs/poetry.lock
 compile_commands.json
 .ccls-cache/
+.mypy_cache
```

Comment:
‎This change seems unrelated to this series, and is also included in a different series that you sent for review.